db/migrate/20170303233627_change_col_name.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeColName < ActiveRecord::Migration
+  def change
+    rename_column :things, :created_at, :drain_import_date
```

Comment:
I believe Rails depends on the names `created_at` and `updated_at` in ActiveRecord to keep these timestamps up to date. You can configure it to use different names in the model, but I'm curious what you think of just overriding the label used in the admin section rather than renaming the columns themselves with something like https://github.com/sferik/rails_admin#per-model